required-checks.json

```diff
@@ -92,5 +92,27 @@
       "googleapis/gapic-generator-typescript",
       "googleapis/cloud-profiler-nodejs"
     ]
+  },
+  "ruby": {
```

Comment:
How hard would it be to get this consistent for just a few more of these repos?  :)

Reply:
The checks are the same for signet and auth. I'm open to removing them from the ignore list. I'm doing this to test merge-on-green for google-api-client. I don't know what all this file is used for.

Reply:
I'd be open to this cautious approach (starting with ignoring, and then going wider once we've tested).